### ISSUE TITLE
`brew install urh` not resulting a valid installation of `urh`: `v2.9.3`

### ISSUE THREAD

**Issue by kraktus** [Homebrew/homebrew-core#108986]
Hello, thank you for you time maintaining `brew`. If I can help in any way feel free to ask

### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
$ brew gist-logs urh
Error: No logs.
```

```shell
$ brew config
HOMEBREW_VERSION: 3.5.10-51-g7617109
ORIGIN: https://github.com/Homebrew/brew
HEAD: 7617109d43060aa0225f774bad9971bcbd166e15
Last commit: 7 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 9e386b551d89ceb15da8cf2df3a30c41ff1a0e95
Core tap last commit: 54 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: subl -w
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.1.6 build 1316
Git: 2.32.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.4-arm64
CLT: 13.4.0.0.1.1651278267
Xcode: 13.4.1
Rosetta 2: false
```

```shell
$ brew doctor
Your system is ready to brew.
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Trying to install `urh`: https://github.com/jopohl/urh and launch the application

### What happened (include all command output)?

```
$ urh
Traceback (most recent call last):
  File "/opt/homebrew/bin/urh", line 33, in <module>
    sys.exit(load_entry_point('urh==2.9.3', 'console_scripts', 'urh')())
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/urh/main.py", line 100, in main
    from urh.controller.MainController import MainController
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/urh/controller/MainController.py", line 15, in <module>
    from urh.controller.dialogs.OptionsDialog import OptionsDialog
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/urh/controller/dialogs/OptionsDialog.py", line 16, in <module>
    from urh.dev.native import ExtensionHelper
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/urh/dev/native/ExtensionHelper.py", line 10, in <module>
    from setuptools import Extension
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/setuptools/__init__.py", line 16, in <module>
    import setuptools.version
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/setuptools/version.py", line 1, in <module>
    import pkg_resources
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3260, in <module>
    def _initialize_master_working_set():
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3234, in _call_aside
    f(*args, **kwargs)
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3272, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/pkg_resources/__init__.py", line 581, in _build_master
    ws.require(__requires__)
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/pkg_resources/__init__.py", line 909, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/opt/homebrew/Cellar/urh/2.9.3_2/libexec/lib/python3.10/site-packages/pkg_resources/__init__.py", line 795, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'cython' distribution was not found and is required by urh
```

Note that **I did install cython with `brew install cython`** afterwards, and it did not change the error

### What did you expect to happen?

Start the application, as stated in the repo doc:
https://github.com/jopohl/urh#Installation
`Type urh in a terminal to get it started.`

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
$ brew install urh
$ urh
```

**Comment by cho-m** [comment]
Probably since `libcython` formula isn't added into search path for `urh`'s virtualenv. Only linked formulae don't need any extra work but `libcython` is keg-only.

This may need to add a .pth file into virtualenv to find `libcython` site-packages directory.

**Comment by branchvincent** [comment]
Fixed via #108991

**Comment by kraktus** [comment]
Thank you!
